### DISCUSSION TITLE
Layer, layers and... Layers... 😅

### DISCUSSION THREAD

**Discussion by ExperienceRO** [new-sankaku/stable-diffusion-webui-simple-manga-maker#38] [Ideas]
Hi, yeah, it's me again... Sorry..
How are you??

So, let me whining about something... 😅
The balloons plus texts... Is possible to link the balloon and the text?? It would be easier if I could drag the group (like in Photoshop) instead of each one per time..

Also, groups in a whole cold be something very useful.

Thank you and have a great day!!

**Comment by new-sankaku** [comment]
OK, I'll think about whether to make it possible to select multiple items by holding down the Ctrl key, or to group the speech bubbles and text. What I want to do is to group the speech bubbles and text and dynamically change the size of the text, but it's not that easy...
